src/com/gh4a/fragment/LoadingListFragmentBase.java

```diff
@@ -54,6 +57,13 @@ protected View onCreateContentView(LayoutInflater inflater, ViewGroup parent) {
                     UiUtils.resolveDrawable(getActivity(), R.attr.listBackground));
         }
 
+        mFastScroller = (RecyclerFastScroller) view.findViewById(R.id.fast_scroller);
+        mFastScroller.attachRecyclerView(mRecyclerView);
+
+        int color = UiUtils.resolveColor(view.getContext(), R.attr.colorControlNormal);
+        color = Color.argb(93, Color.red(color), Color.green(color), Color.blue(color));
```

Comment:
Is there a specific reason not to use an XML defined color here? I think I'd prefer that.